python/ray/tests/test_basic.py

```diff
@@ -1314,35 +1313,33 @@ def test_get_dict(ray_start_regular):
 
 
 def test_get_with_timeout(ray_start_regular):
-    def random_path():
-        return os.path.join(tempfile.gettempdir(), uuid.uuid4().hex)
+    @ray.remote(num_cpus=0)
+    class Signal:
```

Comment:
we should either move this to conftest so all tests can share the same implementation or to ray.utils

Reply:
Ah yeah I did this because I've run into issues with serialization in the past when using imported code in remote functions in pytest. I will try.

Reply:
It might work if you reinitialize the remote class in the individual tests. E.g., `Signal = ray.remote(ray.utils.Signal)`

Reply:
Seems to be working! Will push the change.